### DISCUSSION TITLE
How can I tell where vulnerability is coming from

### DISCUSSION THREAD

**Discussion by mickgr** [aquasecurity/trivy#1648] [Q&A] [answered]
Hi,
I am running image scan and getting 1 vulnerability for lodash@1.0.2
Searched my entire code base and could not find any reference to this version.
also connected to the image and ran fs scan inside and this scan did not raise this vulnerability.

Is there a way to run the image scan with some flag that will provide more information regarding where the vulnerability is coming from?

**Comment by knqyf263** [comment, marked as answer]
`-f json` shows more details.
